crm.conf.in

```diff
@@ -20,6 +20,25 @@
 ; ignore_missing_metadata = no
 ; report_tool_options =
 
+; obscure_pattern option is the persisent configuration of CLI.
+; Example, for the high security concern,
+;
+;        obscure_pattern = passw* | ip
```

Comment:
Why do we have so many spaces here before `obscure_pattern`?
The other examples are using just 1 space

Reply:
Changed

Reply:
Sorry a late review since I miss the click to submit my reviews. Maybe you can take this feedback in the next change set.

Better to use ip* here, since quit some customers do use:

params ip=****
params ipaddr=****
params ip_address=****